internal/pkg/externalplugins/cherrypicker/cherrypicker_invite.go

```diff
@@ -0,0 +1,68 @@
+package cherrypicker
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"k8s.io/test-infra/prow/github"
+)
+
+func (s *Server) inviteCollaborator(ic *github.IssueCommentEvent) error {
+	org := ic.Repo.Owner.Login
+	repo := ic.Repo.Name
+	prNum := ic.Issue.Number
+	commentUser := ic.Comment.User.Login
+	forkToOwner := s.BotUser.Login
+	forkToRepo := repo
+	forkToFullRepo := fmt.Sprintf("%s/%s", forkToOwner, forkToRepo)
+	log := s.Log.WithFields(logrus.Fields{"org": org, "repo": repo, "prNum": prNum})
+
+	// already collaborator in forked repo.
+	if ok, err := s.GitHubClient.IsCollaborator(forkToOwner, forkToRepo, commentUser); err != nil {
```

Comment:
Don't forget to handle pending collaborator invitations.

https://docs.github.com/en/rest/collaborators/invitations#list-repository-invitations

https://github.com/tidb-incubator/cherry-bot/blob/c9bf8ab525ed059c8f3ec6e7741eeae4565990ba/pkg/providers/cherry/request.go#L774